src/Http/Controllers/Admin/ModuleController.php

```diff
@@ -1024,7 +1034,7 @@ public function show($id, $submoduleId = null)
      * @param int|null $submoduleId
      * @return \Illuminate\Http\JsonResponse|\Illuminate\Http\RedirectResponse|\Illuminate\View\View
      */
-    public function edit(TwillModelContract | int $id, $submoduleId = null)
+    public function edit(TwillModelContract|int $id, $submoduleId = null)
```

Comment:
Was the code style ruleset updated for those stylistic changes to happen on this PR?

Reply:
The previous formatting was incorrect, however, there's no specific rules changed.

Reply:
Sounds good!